src/common/components/bodies.h

```diff
@@ -35,6 +35,7 @@ struct LightEmitter {};
  */
 struct StarSystem {
     std::vector<entt::entity> bodies;
+    std::vector<entt::entity> ships;
```

Comment:
Keep all ships in `bodies`, adding more vectors would only make it a bit confusing. We can always update ships by querying if they are a ship or not.